Gaijinizer patch/Map061.txt

```diff
@@ -198,96 +217,103 @@ E-003を投下する前にあの男が死ねば…
 少なくとも、E-003を解き放つ必要はなくなる。
 そういうこったろ？
 > CONTEXT: Map061/events/5/pages/0/123/Dialogue/other2
-If the man dies before dropping E-003 ...
-At least, there is no need to release E-003.
-Is that what happened?#MTLed
+At the very least, if that man dies before E-003
+gets airdropped in, there'll be no need to release
+it, right?#Ron
```

Comment:
Line limits:
>_**At the very least, if that man dies before E-003
gets airdropped in, there'll be no need to
 release it, right?#Ron**_